pkg/controller/datavolume-controller.go

```diff
@@ -438,6 +438,20 @@ func (r *DatavolumeReconciler) Reconcile(_ context.Context, req reconcile.Reques
 		return reconcile.Result{}, err
 	}
 
+	// When cloning a PVC, if the target's size is not specified,
+	// said value can be attainable from the source PVC
+	if selectedCloneStrategy != NoClone {
+		targetRequest, hasTargetRequest := pvcSpec.Resources.Requests[corev1.ResourceStorage]
+		if hasTargetRequest && targetRequest.IsZero() {
```

Comment:
isn't it also ok that hasTargetRequest will be false? like either:
if !hasTargetRequest || targetRequest.IsZero() {
also asking about line 1670: if !hasSrcRequest || !hasSrcCapacity || !hasTargetRequest {
isnt that ok that !hasTargetRequest in case we have storage API?